com.unity.render-pipelines.universal/Editor/Converter/RenderPipelineConvertersEditor.cs

```diff
@@ -469,6 +471,21 @@ void CheckAllConvertersCompleted()
 
         void InitializeAllActiveConverters(ClickEvent evt)
         {
+            Scene currentScene = SceneManager.GetActiveScene();
+            if (currentScene.isDirty)
+            {
+                if (EditorUtility.DisplayDialog("Scene is not saved.",
+                    "Current scene is not saved. Please save the scene before continuing.", "Save and Continue",
+                    "Cancel"))
+                {
+                    EditorSceneManager.SaveScene(currentScene);
+                }
+                else
+                {
+                    return;
+                }
+            }
+
```

Comment:
Can we extract this in to a function that takes in the Scene, since it's the same code in two places?